### PR TITLE
Fix check cfg output variables

### LINKED ISSUE #387: Check output variables in the check_vlasiator_cfg script
@ykempf

### PR DESCRIPTION
**Pull request by ykempf**
Meh, so much for being lazy yesterday and not making a separate branch.

Anyway, the latter two commits fix #387 by having tools/check_vlasiator_cfg.sh also parse the lists of output and diagnostic variables.

Wait for #388 to have this one here cleaner.

### CODE REVIEW

File: tools/check_vlasiator_cfg.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-mpirun_cmd="aprun -n 1"
-#mpirun_cmd="mpirun -np 1"
+#mpirun_cmd="aprun -n 1"
+mpirun_cmd="mpirun -np 1"
```
[the remark this answers is not in the document]
Comment (ykempf) [reply tools/check_vlasiator_cfg.sh]: Ah yes, sorry, forgot that.

**Comment by ykempf** [comment]
I suppose, indeed, that this feature would be nice. :)

On 25 March 2019 12:07:36 EET, Tuomas <notifications@github.com> wrote:
>tkoskela approved this pull request.
>
>Works ok for me. Can you auto-detect whether aprun is available and use
>mpirun if it isn't?
>
>
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/fmihpc/vlasiator/pull/390#pullrequestreview-218236557

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by ursg** [comment]
Maybe simply using something like this:
```
TEST=$(which aprun && echo "aprun" || echo "mpirun");
echo $TEST
```

**Comment by ykempf** [comment]
Like so?

**Pull request merged**